pkgs/development/tools/database/mermerd/default.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "mermerd";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "KarnerTh";
+    repo = "mermerd";
+    rev = "v${version}";
```

The automated update process works smoother if we try not to get in trouble with branch/tags confusion in upstream repos.

```suggestion
    rev = "refs/tags/v${version}";
```